app/models/user.rb

```diff
@@ -2,4 +2,8 @@ class User < ApplicationRecord
   has_many :posts
   has_many :comments
   has_many :likes
```

Comment:
- [ ] Please include the reference to the foreign key for each of the relationships

Hint
`has_many :posts, foreign_key: 'author_id'`

Reply:
Thanks!